src/noise/mod.rs

```diff
@@ -599,6 +599,10 @@ impl Tunn {
 
         (time, tx_bytes, rx_bytes, loss, rtt)
     }
+
+    pub fn is_expired(&self) -> bool {
```

Comment:
I know I committed it, but it looks out of context somehow

Reply:
It does seem to be used